chaste_codegen/tests/test_rdf.py

```diff
@@ -40,5 +40,6 @@ def test_wrong_params2():
 
 def test_rdf(s_model):
     all_currents = get_variables_transitively(s_model, (OXMETA, 'IonicCurrent'))
-    assert str(all_currents) == ("[_cell$i_Stim, _INa$i_Na, _IKr$i_Kr, _IKs$i_Ks, _Itos$i_tos, _IK1$i_K1, _ICaL$i_CaL,"
-                                 " _INaCa$i_NaCa, _Jrel_SR$j_rel_SR]")
+    assert str(all_currents) == ("[_cell$i_Stim, _INa$i_Na, _INab$i_Nab, _IKr$i_Kr, _IKs$i_Ks, _Itos$i_tos, "
```

Comment:
Do you know why this list has changed?

Reply:
Because the cellml has changed (updated to the latest for this model)